en/PlanView/PlanView.md

```diff
@@ -11,33 +11,39 @@ The image above shows a simple mission which starts with a takeoff, flies throug
 The steps to creating a missions are:
 
 1. Change to Plan View
-2. Add commands to the mission and edit as needed
-3. Send the mission to the vehicle
+2. Add waypoints to the mission and edit as needed
+3. Upload: Send the mission to the vehicle
 4. Change to Fly View and fly your mission
 
 ## Plan Tools
 On the left edge of the screen you will see the Plan Tools. The order of tools from top to bottom is:
 
-* Add Commands
-* [Survey](Survey.md)
+* Add waypoints
+* [Pattern](Survey.md)  ---------------------> change this name ?
 * Sync
 * Center map
-* Map Type
-* Zoom In/Out
+* Zoom In
+* Zoom Out
 
-### Add Commands
-Click to activate the Add Commands tool. While active, clicking in the map will add new mission commands at the clicked location. The tool will stay active until you click it again.
+ > **Note** **Center map**, **Zoom In**, **Zoom Out** are just map visualisation tools that help user view the Plan view map clearly while planning missions. These tools don't affect the mission commands sent to the vehicle in any way.
+
+### Actual vs Planned Home Position
+Home position is the position the vehicle will return to and land on when in Return (to Launch/Home) mode. The actual home position is automatically set to the takeoff position by the autopilot during takeoff. This location can be seen in the Fly view once the user is ready to start the mission. The "Planned Home Position", on the other hand, is merely a virtual home position shown in Plan view to help user plan a mission offline while not connected to the vehicle. "Planned Home Position" only has visual significance in QGC such that the QGC is able to draw waypoint lines correctly to the first actual waypoint. However, when the user goes out on the field and starts the mission, the autopilot uses the actual takeoff position as its home position instead of the "Planned Home Position".
+
+
+### Add Waypoints
+Click to activate the Add Waypoints tool. While active, clicking in the map will add new mission waypoint at the clicked location. The tool will stay active until you click it again. Once you have added a waypoint, you can select it by clicking on it and then drag it around to change its position.
```

Comment:
Tool presses are usually highlighted in bold and tool names themselves are usually highlighted in italic. 

So in this case you would write: "Click to activate the **Add Waypoints** tool." because you're talking about a clicking action. You could alternatively write: Activate the *Add Waypoints* tool by clicking it.